dynawo/sources/Models/CPP/ModelNetwork/DYNModelStaticVarCompensator.cpp

```diff
@@ -45,48 +45,23 @@ namespace DYN {
 
 ModelStaticVarCompensator::ModelStaticVarCompensator(const shared_ptr<StaticVarCompensatorInterface>& svc) :
 Impl(svc->getID()),
-Statism_(0.01),
-kG_(4),
-kP_(1),
-Ti_(0.005),
 stateModified_(false) {
   // init data
-  vNom_ = svc->getVNom();
-  vSetPoint_ = svc->getVSetPoint();
-  double coeff = vNom_ * vNom_ / SNREF;
-  bMin_ = svc->getBMin() * coeff;
-  bMax_ = svc->getBMax() * coeff;
   connectionState_ = svc->getInitialConnected() ? CLOSED : OPEN;
   mode_ = svc->getRegulationMode();
-  isRunning_ = (mode_ == StaticVarCompensatorInterface::RUNNING_V);
-  isStandBy_ = false;
-  bShunt_ = 0;
-
-  if (svc->hasStandbyAutomaton()) {
```

Comment:
Does this mean that the StandByAutomaton is now useless in DataInterface?

Reply:
No (if I remember right), because the extension is still used to set the original regulation mode in the Modelica model and to retrieve the bShunt attribute in importStaticParameters in the data interface class.